test/integration/sign-out-not-signed-in-test.js

```diff
@@ -0,0 +1,28 @@
+var store = require('humble-localstorage')
+var test = require('tape')
+
+var Account = require('../../index')
+
+var baseURL = 'http://localhost:3000'
+
+test('sign out without being signed in', function (t) {
+  t.plan(2)
+
+  // simulate user who has not yet signed in
+  store.clear()
+  store.setObject('account', {})
+
+  var account = new Account({
+    url: baseURL,
+    id: 'abc4567'
```

Comment:
This question ties in with my last one.  Should I remove the `id` from `account`, since (I think) the `id` represents a session ID that would be assigned to a logged-in user?